V7_MIGRATION.md

```diff
@@ -68,3 +69,10 @@ For the App Switch flow, you must update your `info.plist` with a simplified URL
 ## PayPal Native Checkout
 The PayPal Native Checkout integration is no longer supported. Please remove it from your app and 
 use the [PayPal (web)](https://developer.paypal.com/braintree/docs/guides/paypal/overview/ios/v6) integration.
+
+## American Express
+Update initializer for `BTAmericanExpressClient`:
+```diff
+-  var amexClient = BTAmericanExpressClient(apiClient: apiClient)
++   var amexClient = BTAmericanExpressClient(authorization: authorization)
+```
```

Comment:
🥹 _tears of joy_

Nit:
```suggestion
```diff
-  var amexClient = BTAmericanExpressClient(apiClient: apiClient)
+   var amexClient = BTAmericanExpressClient(authorization: "<CLIENT_AUTHORIZATION>")
```
```